rest-api-spec/src/main/resources/rest-api-spec/test/indices.analyze/10_analyze.yaml

```diff
@@ -75,7 +75,7 @@ setup:
 "Detail response with Analyzer":
     - do:
         indices.analyze:
-          body: {"text": "This is troubled", "analyzer": standard, "explain": true}
+          body: {"text": "This is troubled", "analyzer": standard, "explain": "true"}
```

Comment:
Of course, both works. Next test case uses true.
https://github.com/elastic/elasticsearch/pull/16977/files#diff-7b60c5abb786ffd29a08a764da0360efL88